crates/tauri-bundler/src/bundle/windows/nsis/mod.rs

```diff
@@ -650,6 +744,10 @@ fn generate_resource_data(settings: &Settings) -> crate::Result<ResourcesMap> {
     }
     added_resources.push(resource_path.clone());
 
+    if settings.can_sign() {
+      try_sign(&resource_path, settings)?;
```

Comment:
?

Reply:
you already identified the code so are you open to creating a PR as well?